doc/src/stdlib/simd-types.md

```diff
@@ -10,7 +10,7 @@ end
 ```
 
 It has a special compilation rule: a homogeneous tuple of `VecElement{T}` maps to an LLVM `vector`
-type when `T` is a bitstype and the tuple length is in the set {2-6,8-10,16}.
+type when `T` is a bits type and the tuple length is in the set {2-6,8-10,16}.
```

Comment:
Does it have to be a julia `primitive` or would other `isbits` types work? If the former, why wasn't it replaced in the renaming PR?